### PR TITLE
feat: SDA-2705 Add possibility for panning

### PR DESCRIPTION
**Pull request by psjostrom**
## Description
This PR adds the possibility for panning in the screen snippet annotate tool

[SDA-2705](https://perzoinc.atlassian.net/browse/SDA-2705)

Notes: Add possibility for panning

### CODE REVIEW

File: spec/__snapshots__/snippingTool.spec.tsx.snap [hunk 1]
```diff
@@ -62,6 +62,13 @@ exports[`Snipping Tool should render correctly 1`] = `
       className="imageContainer"
     >
       <AnnotateArea
+        annotateAreaDimensions={
+          Object {
+            "height": 600,
```
Comment (johankwarnmarksymphony) [inline spec/__snapshots__/snippingTool.spec.tsx.snap:67]: Why not 800 ?

**Review by johankwarnmarksymphony — CHANGES_REQUESTED** [review]

File: spec/__snapshots__/snippingTool.spec.tsx.snap (hunk 1 above)
Comment (psjostrom) [reply spec/__snapshots__/snippingTool.spec.tsx.snap:67]: This is just the markup for a snapshot test. It is created automatically by Jest to be a copy of the HTML structure in a given time. The reason why it is 600 though is that when seting up state for the tool, the default values are:

  const [annotateAreaDimensions, setAnnotateAreaDimensions] = useState({
    height: 600,
    width: 800,
  });

File: src/app/window-handler.ts [hunk 2]
```diff
@@ -1019,11 +1021,15 @@ export class WindowHandler {
     this.moveWindow(this.snippingToolWindow);
     this.snippingToolWindow.setVisibleOnAllWorkspaces(true);
 
+    this.snippingToolWindow.webContents.openDevTools();
```
Comment (VishwasShashidhar) [inline src/app/window-handler.ts:1024]: is this by mistake? :)

**Review by VishwasShashidhar — CHANGES_REQUESTED** [review]
@psjostrom Please see comment :)

File: src/app/window-handler.ts (hunk 2 above)
Comment (psjostrom) [reply src/app/window-handler.ts:1024]: It's not a bug it's a feature 😶

**Review by VishwasShashidhar — APPROVED** [review]

**Review by johankwarnmarksymphony — APPROVED** [review]

**Pull request merged**